src/elements/featured-videos.html

```diff
@@ -113,44 +113,48 @@
     </style>
 
     <div class="container">
-      <div class="videos-header">
-        <h4 class="heading">{$ featuredVideosTitle $}</h4>
-        <div class="controls">
-          <paper-icon-button icon="icons:chevron-left" on-tap="shiftContentLeft"></paper-icon-button>
-          <paper-icon-button icon="icons:chevron-right" on-tap="shiftContentRight"></paper-icon-button>
+      <template is="dom-if" if="[[_videos]]">
```

Comment:
We can move all paths to firebase db into the config file. Like:
```
firebase: {
  ...
  paths: {
     video: '/video',
     sessions: '/sessions'
     ...
  }
...
```

then if path not defined in config then don't render `firebase-query` in [app-data](https://github.com/gdg-x/hoverboard/blob/master/src/elements/app-data.html) with all corresponding block among a whole app.

Can you do it like that?